### ISSUE TITLE
The behavior of GrabDragInteraction appears to have changed in an undesirable way

### ISSUE THREAD

**Issue by jbphet** [phetsims/scenery-phet#636]
In the published version of Friction (v1.5.10), if the user drags a book with the mouse and then uses the keyboard to navigate to the same book, the grab cue is still shown.  In the current master version, the grab cue is _not_ shown for the same sequence of events.  This was discovered when working to add similar grab cue behavior to Faraday's Law in https://github.com/phetsims/faradays-law/issues/174.  The design team discussed the behavior, and decided that what we're seeing in the published version of Friction is what we want.

I compared `GrabDragInteraction` in the published version of Friction to that in master and found the a press listener has been added since the publication of Friction that I believe is treating grabs from mouse/touch events the same as those that come from the keyboard.  I discussed this with @jessegreenberg over Zoom and he agrees that this is likely where the problem was introduced.  He agreed to see if he could come up with a way to make it behave like it used to, and had several ideas on how to approach it.

**Comment by jessegreenberg** [comment]
This was done in the above commit. I removed `successfulDrag` and `grabsToCue` and replaced with two functions `showDragCueNode` and `showGrabCueNode` which are used instead when checking whether these Nodes should be displayed.

@zepumph can you please review the changes?

**Comment by zepumph** [comment]
Your changes look really good. I felt like we could factor things out into a single function. What do you think?

**Comment by jessegreenberg** [comment]
Yep, that is great thanks! Tested that it still works in faradays-law and friction. Since that was the only comment and @jbphet verified this is working in https://github.com/phetsims/faradays-law/issues/174 I think this can be closed.
